controller/src/functions/renderings-bus.ts

```diff
@@ -122,6 +130,16 @@ export async function wacsSbRenderingsApi(
             `Failed to create new content row for ${`${parsed.User}/${parsed.Repo}`.toLowerCase()}`
           );
         }
+      } else {
+        // some properties we want to upsert on becuase PORT does not track them, such as the title, so if they've changed coming from the renderings bus, we need to upsert them.
+        await upsertContentFromRenderingBus({
+          existingId: currentExistingId,
+          payload: {
+            // as needed: if wacs becomes source of truth over port for some items, or tracks some stuff that port doesn,'t can add that here
+            title: parsed.ResourceName || null,
+          },
+          db,
+        });
```

Comment:
Like above branch of code, becuase `content` can be created in port of from the rendering pipeline, this is an upsert of just the properties that the rendering pipeline is aware.  If there were anything else in manifest we wanted to track that Port didn't, it could go here